src/map/index.js

```diff
@@ -10,22 +10,35 @@ const map = new mapboxgl.Map({
   style: 'mapbox://styles/mapbox/light-v10',
   center: [115.817465, -31.976629],
   zoom: 17,
-  pitchWithRotate: false,
-  dragRotate: false
+  pitchWithRotate: true,
+  dragRotate: true
```

Comment:
While this is pretty cool and on an actual map I'd be ok with it, having it on a tiny map isn't really necessary and if used accidentally would really disorientate someone. I think we'll just leave these off.